drivers/cc110x/cc110x-netdev.c

```diff
@@ -103,15 +103,15 @@ static int _get(netdev_t *dev, netopt_t opt, void *value, size_t value_len)
         case NETOPT_CHANNEL:
             assert(value_len > 1);
             *((uint16_t *)value) = (uint16_t)cc110x->radio_channel;
-            return 2;
+            return sizeof(uint16_t);
         case NETOPT_ADDRESS:
             assert(value_len > 0);
             *((uint8_t *)value) = cc110x->radio_address;
             return 1;
```

Comment:
what about this line?

Reply:
Addresses for the cc11xx are 1 byte long, so I'm not sure I understand your unspecific question ;)

Reply:
So, why not use `sizeof(cc110x->radio_channel)` here? Or `sizeof(uint8_t)`? At least this seemed your motiviation when I look at your change a few lines above.

Reply:
Typical "while you're at it" comment, but fine... will fix.